src/test/resources/gradle_plugin/pipeline_test_build_scan.txt

```diff
@@ -0,0 +1,12 @@
+stage ('build_Project') {
+    steps {
+        script {
+           if(isUnix()) {
```

Comment:
But the tool command is required, or not? Otherwise grade will not be installed.